project/CelebornBuild.scala

```diff
@@ -872,3 +882,82 @@ trait FlinkClientProjects {
       )
   }
 }
+
+////////////////////////////////////////////////////////
+//                   MR Client                        //
+////////////////////////////////////////////////////////
+object MRClientProjects {
+
+  def mrClient: Project = {
+    Project("celeborn-client-mr", file("client-mr/mr"))
+      .dependsOn(CelebornCommon.common % "test->test;compile->compile")
+      .dependsOn(CelebornClient.client % "test->test;compile->compile")
+      .settings(
+        commonSettings,
+        libraryDependencies ++= Seq(
+          Dependencies.hadoopClientApi,
+          Dependencies.hadoopClientRuntime,
+          Dependencies.hadoopMapreduceClientApp
+        ) ++ commonUnitTestDependencies
+      )
+  }
+
+  def mrClientShade: Project = {
+    Project("celeborn-client-mr-shaded", file("client-mr/mr-shaded"))
+      .dependsOn(mrClient)
+      .settings(
+        commonSettings,
+
+        // align final shaded jar name with maven.
+        (assembly / assemblyJarName) := {
+          val extension = artifact.value.extension
+          s"${moduleName.value}_${scalaBinaryVersion.value}-${version.value}.$extension"
+        },
+
+        (assembly / test) := {},
+
+        (assembly / logLevel) := Level.Info,
+
+        // Exclude `scala-library` from assembly.
```

Comment:
I think we need to include the `scala-library`.
https://github.com/apache/incubator-celeborn/blob/8eba1b470e9cd54cf0cdcefc381433ce94d881a0/client-mr/mr-shaded/pom.xml#L62-L65

Reply:
It seems that this `relocation` does not take effect.

Reply:
then shall we remove this rule?